plugins/com.google.cloud.tools.eclipse.appengine.localserver.test/src/com/google/cloud/tools/eclipse/appengine/localserver/ui/AppEngineTabGroupTest.java

```diff
@@ -26,7 +27,10 @@ public void testCreateTabs() {
     AppEngineTabGroup group = new AppEngineTabGroup();
     group.createTabs(null, "");
     Assert.assertEquals("Server", group.getTabs()[0].getName());
-    Assert.assertEquals("Environment", group.getTabs()[1].getName());
+    Assert.assertEquals("Arguments", group.getTabs()[1].getName());
+    for (ILaunchConfigurationTab tab : group.getTabs()) {
```

Comment:
Shouldn't this go ahead of line 29, since you're dereferencing the objects to call `getName()`?